src/utils/helperFunctions.js

```diff
@@ -66,3 +66,16 @@ export const calculateTimeStamp = time => {
   const date = new Date(time);
   return date.toDateString();
 };
+
+/**
+ *
+ * @param {object} nextProps next props of an article to edit
+ * @param {object} currentProps this.props of the article component
+ */
+
+export const checkNextProps = (nextProps, currentProps) => {
+  if (isEqual(nextProps, currentProps)) {
```

Comment:
Why are you using lodash here?
what do you think about using ES6 only?

Reply:
@espoirMur I don't know if you clearly checked what is being compared. These are objects which might have nested fields, so think of loops or maps that will have to be written to iterate through  two objects, with comments, tags, etc., to compare two different objects.